### PR TITLE
8304761: Update IANA Language Subtag Registry to Version 2023-03-22

### PR DESCRIPTION
**Pull request by justin-curtis-lu**
Update IANA LSR to 2023-03-22 [update](https://mm.icann.org/pipermail/ietf-languages-announcements/2023-March/000085.html). Add all new preferred languages to `ACCEPT_LANGUAGES` except `gu`.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8304761](https://bugs.openjdk.org/browse/JDK-8304761): Update IANA Language Subtag Registry to Version 2023-03-22

### Reviewers
 * [Naoto Sato](https://openjdk.org/census#naoto) (@naotoj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13165/head:pull/13165` \
`$ git checkout pull/13165`

Update a local copy of the PR: \
`$ git checkout pull/13165` \
`$ git pull https://git.openjdk.org/jdk.git pull/13165/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13165`

View PR using the GUI difftool: \
`$ git pr show -t 13165`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13165.diff">https://git.openjdk.org/jdk/pull/13165.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/util/Locale/LanguageSubtagRegistryTest.java [hunk 1]
```diff
@@ -163,14 +177,22 @@ public class LanguageSubtagRegistryTest {
...
+            new LanguageRange("tpw", 1.0),
             new LanguageRange("tvx", 1.0),
+            new LanguageRange("umi", 1.0),
+            new LanguageRange("szd", 1.0),
             new LanguageRange("uss", 1.0),
             new LanguageRange("uth", 1.0),
             new LanguageRange("ysm", 1.0),
             new LanguageRange("sgn-ysm", 1.0),
-            new LanguageRange("wkr", 0.9),
+            new LanguageRange("wkr", 1.0),
```
[the remark this answers is not in the document]
Comment (justin-curtis-lu) [reply test/jdk/java/util/Locale/LanguageSubtagRegistryTest.java]: `wkr` added into `ACCEPT_LANGUAGES` incorrectly by me, fixed now.

**Review by naotoj — APPROVED** [review]
LGTM

**Pull request closed without merging**